wagtail_localize/templates/wagtail_localize/modeladmin/includes/button.html

```diff
@@ -0,0 +1 @@
+<a{% if button.url %} href="{{ button.url }}{% if locale %}?locale={{ locale.language_code }}{% endif %}"{% endif %} class="{{ button.classname }}" title="{{ button.title }}"{% if button.target %} target="{{ button.target }}"{% endif %}{% if button.rel %} rel="{{ button.rel }}"{% endif %}>{{ button.label }}</a>
```

Comment:
Could we move these templates into the ``wagtail_localize.modeladmin`` app?

Reply:
Yes, that makes sense, moved it!